tests/bootstrap.php

```diff
@@ -4,11 +4,11 @@
 	die( 'Not an entry point' );
 }
 
-error_reporting( E_ALL | E_STRICT );
-ini_set( 'display_errors', 1 );
+error_reporting( -1 );
+ini_set( 'display_errors', '1' );
```

Comment:
What was wrong with `1` being an integer?

Reply:
Fails when strict_types in PHP 7